### ISSUE TITLE
error The token provided must not be empty

### ISSUE THREAD

**Issue by javialon26** [jsnanigans/vue-parallax-js#8]
Hello, with vue 2.5.13 and nuxt next version i get this error:

```
Failed to execute 'add' on 'DOMTokenList': The token provided must not be empty.
An error occurred while rendering the page. Check developer tools console for details.
```

```
client.js?a0f2:52 DOMException: Failed to execute 'add' on 'DOMTokenList': The token provided must not be empty.
    at ParallaxJS.add (webpack-internal:///./node_modules/vue-parallax-js/lib/vue-parallax-js.es.js:1:657)
    at inserted (webpack-internal:///./node_modules/vue-parallax-js/lib/vue-parallax-js.es.js:1:1636)
    at callHook$1 (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:6192:7)
    at callInsert (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:6131:9)
    at _update (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:6137:7)
    at Array.updateDirectives (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:6095:5)
    at patchVnode (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:5830:62)
    at updateChildren (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:5721:9)
    at patchVnode (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:5835:29)
    at updateChildren (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:5721:9)
```

example of html code:
```
<div class="slider-inner swiper-home" v-parallax="0.1">
...
```

before it worked well with vue 2.4.2 and nuxt 1.0.0-rc11

**Comment by jsnanigans** [comment]
Thanks for reporting this, I'll look into it as soon as possible

**Comment by piecioshka** [comment]
@javialon26 I've the same error, and resolve them by setting `className: 'xx'` ex.

```
import Vue from 'vue';
import VueParallaxJs from 'vue-parallax-js';

Vue.use(VueParallaxJs, {
    className: 'movement-parallax'
});
```

@jsnanigans You should probably set default className but it should't be empty.

**Comment by sijn82** [comment]
Thanks @piecioshka I've just had the same error and your solution worked for me as well.

**Comment by jsnanigans** [comment]
Hey, sorry it took me so long to get to this.
This error is fixed in v0.0.7 which is now live. Thanks for reporting this issue
